### ISSUE TITLE
nvda crashes after logon screen in windows 10

### ISSUE THREAD

**Issue by alexandretoco** [nvaccess/nvda#13694]
### Steps to reproduce:
I just type my pin at the logon screen. After a while nvda loads and crash and load again.
### Actual behavior:
nvda crashes after the logon screen and restart it self. it causes the windows's configuration screen to show up, the same as if I press windows key + i. 

### Expected behavior:
nvda should load and don't crash
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2022.1.rc1 
#### Windows version:
windows 10
#### Name and version of other software in use when reproducing the issue:
sapi 5 eloquence
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I didn't test this
[nvda-old.log](https://github.com/nvaccess/nvda/files/8686132/nvda-old.log)

**Comment by lukaszgo1** [comment]
cc @michaelDCurran

**Comment by josephsl** [comment]
Hi, does this happen when you use a different speech synthesizer when starting NVDA? If not, then it looks something odd is going on with SAPI5 driver, as the failure has to do with the fact that by the time SAPI5 Eloquence speaks, wx.app is not even ready yet. Thanks.

**Comment by alexandretoco** [comment]
You got the point!

I changed the synthesizer to one core voices, saved the profile and 
restart. This time the crash didn't occur.

But, when I start with sapi 5 eloquence, I hear the nvda's start sound, 
after I hear the message "loading nvda please wait" with eloquence and 
after it crashes.

So, if the synthesizer was not loaded, how can I hear this first message?


Em 13/05/2022 07:24, Joseph Lee escreveu:

**Comment by lukaszgo1** [comment]
Can we consider creating our instance of `wx.App` earlier in `core.main`? This should be safe as  far as I can tell. In the long run it would be nice to change `core.callLater` not to rely on  app instance being created and / or make sapi5 driver use `nvwave` directly, but both of these are way too risky for 2022.1 at this stage IMO.

**Comment by CyrilleB79** [comment]
For people investigating, one thing to note also in the log:
```
DEBUGWARNING - core.main (06:31:26.214) - MainThread (13496):
Slow starting core (19.04 sec)
```

**Comment by Brian1Gaff** [comment]
There is a long running issue where it does not crash,but comes up with no 
sound and if you  look at theold log after a restart it says there is no 
sound device found. I've had this happen on Windows 7 and 10. It is usually 
due to a lot of things trying to load at the start and not quite making it 
in time for nvda to see the audio drivers are loaded and initialized, but 
obviously, this will not allow you to use it on the start screen either to 
log in of course. The answer seems to be often solved by decluttering start 
up by removing the  crap, or sometimes by Ccleaner  being run.
  I don't think I've seen it happen as described where it runs, but falls 
over. one would imagine that if it is speaking on the log in it should speak 
everywhere. What synth iis actually used for log in? If its different from 
that used in the main program, I can see that might cause an issue. When I 
get sound issues I tend to set everything to Espeak and save the config, 
then  try it  a few times at start up.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Brian1Gaff** [comment]
Hmm, but in some cases this has occurred for several versions to me at 
least, and decluttering did seem to resolve it. I wonder if something else 
is also using Sappi 5, it was in my case, but I gave up on that due to lag.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Brian1Gaff** [comment]
Yes could the machine be one of those with a normal drive, say running a 
slowprocessor or have low ram? It is obvious to me at least that some 
machines being sold even now only have 4gigs of ram, You need 8 or greater 
for smooth boot up.
Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by seanbudd** [comment]
@alexandretoco  Can you please test and confirm this is not an issue with 2021.3.5?
